testing/test_unittest.py

```diff
@@ -718,3 +718,16 @@ def test_one(self):
         *SKIP*[1]*test_foo.py*skipping due to reasons*
         *1 skipped*
     """)
+
+def test_unittest_skip_issue1169(testdir):
+    testpath = testdir.makepyfile(test_foo="""
+        import unittest
+        
+        class MyTestCase(unittest.TestCase):
+            @unittest.skip
+            def test_skip(self):
+                 self.fail()
+        """)
+    reprec = testdir.inline_run(testpath)
+    reprec.assertoutcome(passed=1)
```

Comment:
OK I am laughing. skipped=1 was my first guess. I think it is the outcome of the test case rather than the test case's methods. Arguably, a second test case would be nice, testing skipping a bare test function and that one might have an outcome of skipped=1.